components/webflow/actions/create-collection-item/create-collection-item.mjs

```diff
@@ -1,53 +1,59 @@
-// legacy_hash_id: a_q1ioGv
-import { axios } from "@pipedream/platform";
+import webflow from "../../webflow.app.mjs";
 
 export default {
   key: "webflow-create-collection-item",
-  name: "Create Item",
+  name: "Create Collection Item",
   description: "Create new collection item",
```

Comment:
I think you can add the docs like
```suggestion
  description: "Create new collection item. [See the docs here](https://developers.webflow.com/#create-new-collection-item)",
```
and the same goes for the rest of components that don't have any doc reference in the `description` property